float32 rotateDelta
---
int32 result
